middle_value.js

```diff
@@ -0,0 +1,12 @@
+function findMiddleValue (array) {
+  if (array instanceof (Array) == false || array.length == 0) {
```

Comment:
please separate statements and iterators with break-lines

Reply:
I'm not sure that I understood it correctly, but I tried. Done